consts.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+const (
+	tempRepoPath = "./temp_repo"
+)
```

Comment:
We want request concurrency. Using one file path is racy. You need to create a new random temp folder for each request (there probably is a library for this)

Maybe we can do this with the other fix later

Reply:
will add this to the issue